### ISSUE TITLE
Blue Screen 2 or 10 minutes after opening application

### ISSUE THREAD

**Issue by FunnyRAGE** [artman41/guitarsniffer#14]
I can play maybe one song, or get thorugh half of the song and the application will full on crash my pc.

**Comment by jesusandthepope** [comment]
Not very helpful for solving the issue but maybe a little comfort. My friend and I have been experiencing the same issue then crash to blue screen dpc watchdog violation. This issue only arises while we are using guitar sniffer. Hopefully someone figures it out but that's all I can contribute.

**Comment by Dunkalunk** [comment]
Wow...  I have no idea why that would be happening.  I'm not playing much these days but I got on this morning and played for half an hour or so with no issues.
Only question I've thought of is if you're running the program as Admin or not?  If not I would give that a try.  Otherwise I have no idea.

**Comment by jesusandthepope** [comment]
Running as admin and ran a diag tool not getting above 25% cpu (i5 4 core t1050). Crashes with and without diag running never spikes gpu or cpu or memory to max. Just blue screen.
Also have uninstalled and reinstalled the programs usb, win, vjoy, and sniffer to no success.
16g dual channel ram 35-40% usage.
GPU 2 gig vram 28%
CPU 22% on most recent crash(blue screen dpc watchdog)
500g ssd with up to date drivers running windows 10
Thought some specs might help.

**Comment by jesusandthepope** [comment]
Btw other than crashing thing works perfect 10/10. Low latency full tracking of tilt, whammy, and all frets. Can be run without the Vjoy, usb, and win program, if you replace those with a controller app like antimicro. But still blue screen. I tried alot of junk and taught myself alot trying to get around the issue but hey learning to program seems beyond me but I'm pretty stubborn so that might be the rout I have to take.

**Comment by Dunkalunk** [comment]
Yeah and to add on top of all the trouble, I'm not even the creator...  He went MIA months ago.  I took what he did for the guitar side and mimicked it for the drums.  So I'm just trying to be as helpful as I can but this is probably beyond me as well :(

**Comment by FunnyRAGE** [comment]
> Wow... I have no idea why that would be happening. I'm not playing much these days but I got on this morning and played for half an hour or so with no issues.
> Only question I've thought of is if you're running the program as Admin or not? If not I would give that a try. Otherwise I have no idea.

Yea. I always run as administrator, still the problem occurs
